components/scream/src/dynamics/homme/atmosphere_dynamics.cpp

```diff
@@ -287,7 +287,7 @@ void HommeDynamics::initialize_impl (const util::TimeStamp& /* t0 */)
   const auto& Q_dyn_track = get_group_out("Q",dgn).m_bundle->get_header().get_tracking();
   for (auto wp : Q_dyn_track.get_providers()) { 
     auto sp = wp.lock();
-    printf("Q is provided by: %s\n",sp->name().c_str());
+    if (get_comm().am_i_root()) printf("Q is provided by: %s\n",sp->name().c_str());
```

Comment:
I think I added this for loop while I was debugging (trying to figure out who else was computing Q). Can you please remove it altogether?